Simplenote/NSObject+Simplenote.m

```diff
@@ -14,7 +14,7 @@ @implementation NSObject (Simplenote)
 
 + (BOOL)sp_respondsToSelector:(SEL)aSelector
 {
-    return class_respondsToSelector([self class], aSelector);
+    return [self respondsToSelector:aSelector];
```

Comment:
For some reason, this check wasn't working properly unless I used `[NSObject respondsToSelector]`